regex-assembly/933160.ra

```diff
@@ -8,7 +8,7 @@
 ##!^ ['\"]*
 
 ##! match comments: `/*...*/`, `//...`, `#...`
-##!$ (?:/\*.*\*/|//.*|#.*
+##!$ (?:/\*[^*][^/]*\*/|//[^\r\n]*|#[^\r\n]*
```

Comment:
Sorry, that was BS. I don't actually see a good way to remove `.*` in this instance.
```suggestion
##!$ (?:/\*.*\*/|//[^\r\n]*|#[^\r\n]*
```

Reply:
Yes, why not.